### PR TITLE
Show filename when overflowing

### PR DESCRIPTION
**Pull request by mrfyda**
screen shot 2016-05-22 at 10 11 20

### CODE REVIEW

**Comment by mrfyda** [comment]
@rtfpessoa it appears circleci is not running for my PRs... because it's a fork?

**Comment by rtfpessoa** [comment]
It is weird. All the previous prs from other people worked.

I will look a bit in the docs.

**Comment by rtfpessoa** [comment]
I forced the run for unsafe builds from forks. But now I do not seem to be able to trigger a build from your fork.
Can you close and open the pr again?

**Comment by rtfpessoa** [comment]
Regarding the code, since you used a new method for preparing the name, the names on the file summary are still being displayed in the old way. What I suggest is for you to return an object like:

```
{
path: "my/path"
name: "file.js"
}
```

And use `getDiffName` as the main method. Maybe move the logic on `getDiffName` to other private method.


Other thing I think that might be interesting is to give one more sub-path for context, ex:

Full path: `src/with/a/really/long/path/my/file.js`

would be split into:

Path prefix: `src/with/a/really/long/path/`
Path suffix: `my/file.js`

What do you think?

File: src/ui/css/diff2html.css [hunk 1]
```diff
@@ -246,7 +246,7 @@
   text-align: left;
 }
 
-.d2h-file-list-line .d2h-file-name {
+.d2h-file-list-line .d2h-file-path {
```
Comment (rtfpessoa) [inline src/ui/css/diff2html.css]: sasasasas

Comment (rtfpessoa) [inline src/ui/css/diff2html.css]: qqqqqq

**Comment by rtfpessoa** [comment]
@mrfyda are planning to fix the issues here?

**Comment by rtfpessoa** [comment]
@mrfyda are planning to fix the issues here?

**Comment by rtfpessoa** [comment]
@mrfyda are planning to fix the issues here?

**Comment by rtfpessoa** [comment]
@mrfyda are planning to fix the issues here?

**Comment by rtfpessoa** [comment]
@mrfyda are planning to fix the issues here?

**Comment by rtfpessoa** [comment]
@mrfyda are planning to fix the issues here?

**Pull request closed without merging**